components/sendgrid/actions/add-email-to-global-supression/add-email-to-global-supression.js

```diff
@@ -0,0 +1,23 @@
+const common = require("../common");
+
+module.exports = {
+  ...common,
+  key: "sendgrid-add-email-to-global-supression",
+  name: "Add Email To Global Supression",
+  description:
+    "Allows you to add one or more email addresses to the global suppressions group.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    recipientEmails: {
+      type: "string[]",
+      label: "Recipient Emails",
+      description:
+        "An string array of email addresses to be added to the global suppressions group. Example `[\"email1@example.com\",\"email2@example.com\"]`",
+    },
+  },
+  async run() {
+    return await this.sendgrid.addEmailToGlobalSupression(this.recipientEmails);
```

Comment:
As far as I know the `await` keyword here is not necessary. Since you are not doing anything with a `try/catch` block. So you can remove the `await` and anywhere else you are using it along with `return` and without using `try/catch`

Reply:
oh yeah I remember we discussed this for Shortcut (formerly Clubhouselio)

> Hi [@sergioeliot2039](https://github.com/sergioeliot2039) I've found [this article](https://jakearchibald.com/2017/await-vs-return-vs-return-await/) very interesting in order to totally understand await vs return vs return await. I hope it helps!

> In [components/shortcut/actions/create-story/create-story.js](https://github.com/PipedreamHQ/pipedream/pull/1383#discussion_r762045467):

For sure, I'll get to this

Reply:
I made this change as requested.

Reply:
I still see it in the code 🤔

Reply:
Right, looks like I oversaw a few of those. But most had been addressed, All of them should be good now.